services/venice-server/src/main/java/com/linkedin/venice/listener/request/CurrentVersionRequest.java

```diff
@@ -12,8 +12,7 @@ private CurrentVersionRequest(String storeName) {
     this.storeName = storeName;
   }
 
-  public static CurrentVersionRequest parseGetHttpRequest(HttpRequest request) {
-    String uri = request.uri();
+  public static CurrentVersionRequest parseGetHttpRequest(URI uri) {
```

Comment:
Can we do the same refactoring as the other functions in this one?

Reply:
Sure, I missed this one.